fuzz/fuzz_targets/table_ops.rs

```diff
@@ -0,0 +1,45 @@
+#![no_main]
+
+use libfuzzer_sys::{fuzz_mutator, fuzz_target, fuzzer_mutate};
+use mutatis::Session;
+use wasmtime_fuzzing::generators::table_ops::TableOps;
+use wasmtime_fuzzing::oracles::fuzz_table_ops;
+
+fuzz_target!(|input: (u64, TableOps)| {
+    fuzz_table_ops(input);
+});
+
+fuzz_mutator!(|data: &mut [u8], size: usize, max_size: usize, seed: u32| {
+    let _ = env_logger::try_init();
+
+    // With probability of about 1/8, just use the default mutator.
+    if seed.count_ones() % 8 == 0 {
+        return fuzzer_mutate(data, size, max_size);
+    }
+
+    // Fallback to default on decode failure
+    let mut tuple =
+        bincode::decode_from_slice::<(u64, TableOps), _>(data, bincode::config::standard())
+            .map_or_else(|_err| (0, TableOps::default()), |(tuple, _)| tuple);
```

Comment:
Slight simplification:

```suggestion
        bincode::decode_from_slice::<(u64, TableOps), _>(data, bincode::config::standard())
            .ok()
            .unwrap_or_default();
```

Reply:
This is nice!